view/checkout.php

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Checkout</title>
+        <link rel="stylesheet" type="text/css" href="css/login_register_checkout.css">
```

Comment:
References to this folder now require it to include `view` in the path name.
E.g instead of `css/login_register_checkout.css` you'd reference `view/css/login_register_checkout.css`